### DISCUSSION TITLE
Allow custom CSS

### DISCUSSION THREAD

**Discussion by iced-queen** [GitbookIO/community#165] [Feature Requests]
### What's your feature/idea?

I think it'd be cool if you allowed _some_ custom CSS.
Like for example to use custom colours in Info Blocks, or change colours in Code Blocks.

### What's your desired outcome?

Well if it was implemented, I'd probably use it to change colours of some things here and there. E.g. in Code Blocks and Info Blocks.

### What's the impact of this for you?

- [ ] Critical
- [ ] High
- [X] Medium
- [ ] Low

**Comment by aikoii** [comment]
Hi @chocobearxo!

Thanks for writing in and sharing your specific case. 

I'm afraid we don't have plans to offer the option to use custom code inside GitBook. That said, we are working on introducing integrations that can be built by third parties, so this could be something to look into at that point! 🙂

**Reply by Reelie15** [reply]
> I'm afraid we don't have plans to offer the option to use custom code inside GitBook. That said, we are working on introducing integrations that can be built by third parties, so this could be something to look into at that point! 🙂

Hi there!

I am currently trying to build an integration that would allow that. Would it be possible in the current state of GitBook?

I'm following the steps here, but am encountering an error at the `gitbook dev` command:
https://developer.gitbook.com/getting-started/development

Am I working for nothing, or I should be able to do it? I've not been able to find a place with public integrations made by other yet, which might indicates that it's not possible yet.

Thanks!
-Reelie

**Reply by valentin0h** [reply]
Hi @Reelie15 ,

I'm sorry to hear you are having issues with integration development. What kind of error is it producing? Perhaps you can paste the output here so we can look. 

We are actively working on improving the DX working with the integration platform.

**Reply by Reelie15** [reply]
Oh well, thanks for the help!

Here's my output:
```
C:\My\Path>gitbook dev myId
\ Starting dev server...
node:events:497
      throw er; // Unhandled 'error' event
      ^

Error: spawn C:\Users\MyUser\AppData\Roaming\npm\node_modules\@gitbook\cli\dist\cloudflared ENOENT
    at ChildProcess._handle.onexit (node:internal/child_process:286:19)
    at onErrorNT (node:internal/child_process:484:16)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
Emitted 'error' event on ChildProcess instance at:
    at ChildProcess._handle.onexit (node:internal/child_process:292:12)
    at onErrorNT (node:internal/child_process:484:16)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21) {
  errno: -4058,
  code: 'ENOENT',
  syscall: 'spawn C:\\Users\\MyUser\\AppData\\Roaming\\npm\\node_modules\\@gitbook\\cli\\dist\\cloudflared',
  path: 'C:\\Users\\MyUser\\AppData\\Roaming\\npm\\node_modules\\@gitbook\\cli\\dist\\cloudflared',
  spawnargs: [ 'tunnel', '--no-autoupdate', '--url', 'http://localhost:52505' ]
}

Node.js v20.18.0
```
I think I've already tried to reinstall NodeJs, but it's been a short while already. I haven't spent so much time investigating it yet.

Shall I create another thread in the forum for this?
